project/Osgi.scala

```diff
@@ -50,7 +50,8 @@ object Osgi {
       "Bundle-SymbolicName" -> (bundleSymbolicName.value + ".source"),
       "Bundle-Version" -> versionProperties.value.osgiVersion,
       "Eclipse-SourceBundle" -> (bundleSymbolicName.value + ";version=\"" + versionProperties.value.osgiVersion + "\";roots:=\".\"")
-    )
+    ),
+    Keys.`package` := bundle.value
```

Comment:
Do the doc and source packages still work correctly? We only want to change `packageBin`.